### PR TITLE
Refactor Setting Internal Configurable Rate Limits

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
Refactor the HTML used to set Rate limits. Lots more refactoring to come but this was a quickie that eliminated a lot of lines that I thought I could separate from the rest. 

## Added tests?
- [x] no, because they aren't needed

alt_text

### CODE REVIEW

File: app/labor/rate_limit_checker.rb [hunk 1]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
 
-  CONFIG_LIMIT_KEYS = %i[
-    rate_limit_follow_count_daily
-    rate_limit_comment_creation
-    rate_limit_published_article_creation
-    rate_limit_image_upload
-    rate_limit_email_recipient
-  ].freeze
+  CONFIGURABLE_RATES = {
+    rate_limit_follow_count_daily: { min: 0, placeholder: 500, description: "Used to limit the amount of users a person can follow daily" },
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:13]:
```suggestion
    rate_limit_follow_count_daily: { min: 0, placeholder: 500, description: "The number of users a person can follow daily" },
```

File: app/labor/rate_limit_checker.rb [hunk 2]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
+    rate_limit_comment_creation: { min: 0, placeholder: 9, description: "Used to limit the amount of comments a user can create within 30 seconds" },
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:14]:
```suggestion
    rate_limit_comment_creation: { min: 0, placeholder: 9, description: "The number of comments a user can create within 30 seconds" },
```

File: app/labor/rate_limit_checker.rb [hunk 3]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
+    rate_limit_published_article_creation: { min: 0, placeholder: 9, description: "Used to limit the amount of articles a user can create within 30 seconds" },
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:15]:
```suggestion
    rate_limit_published_article_creation: { min: 0, placeholder: 9, description: "The number of articles a user can create within 30 seconds" },
```

File: app/labor/rate_limit_checker.rb [hunk 4]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
+    rate_limit_image_upload: { min: 0, placeholder: 9, description: "Used to limit the amount of images a user can upload within 30 seconds" },
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:16]:
```suggestion
    rate_limit_image_upload: { min: 0, placeholder: 9, description: "The number of images a user can upload within 30 seconds" },
```

File: app/labor/rate_limit_checker.rb [hunk 5]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
+    rate_limit_email_recipient: { min: 0, placeholder: 5, description: "Used to limit the amount of emails we send to a user within 2 minutes" },
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:17]:
```suggestion
    rate_limit_email_recipient: { min: 0, placeholder: 5, description: "The number of emails we send to a user within 2 minutes" },
```

File: app/labor/rate_limit_checker.rb [hunk 6]
```diff
@@ -9,13 +9,14 @@ class RateLimitChecker
...
+    rate_limit_organization_creation: { min: 1, placeholder: 1, description: "Number of organizations a user can create within a 5 minute period" }
```
Comment (vaidehijoshi) [inline app/labor/rate_limit_checker.rb:18]:
```suggestion
    rate_limit_organization_creation: { min: 1, placeholder: 1, description: "The number of organizations a user can create within a 5 minute period" }
```

**Comment by Ridhwana** [comment]
Very apt gif too 😁

**Pull request merged**